styles.css

```diff
@@ -74,3 +78,313 @@ body {
   padding: 0;
   padding-bottom: 138px;
 }
+
+/* mid section */
+
+#mid {
+  display: grid;
+  grid-template-columns: 24px 1fr 24px;
+  row-gap: 88px;
+}
+
+.left-block {
+  grid-column: 2;
+  display: flex;
+  flex-direction: column;
+}
+
+.primary-image {
+  border-radius: 8px;
+  margin-top: 16px;
+  margin-left: 16px;
+  margin-right: 16px;
+  margin-bottom: 12px;
+}
+
+.title h1 {
+  margin-bottom: 20px;
+  margin-left: 16px;
+  margin-right: 16px;
+}
+
+.frame-2 {
+  margin-bottom: 20px;
+  margin-left: 16px;
+}
+
+.frame-2 ul {
+  display: flex;
+  padding: 0;
+}
+
+.frame-2 ul li {
+  margin-right: 32px;
+}
+
+.primary-text {
+  margin-left: 16px;
+  margin-right: 16px;
+  margin-bottom: 16px;
+  max-width: 295px;
+  max-height: 76px;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+}
+
+.tags {
+  margin-bottom: 40px;
+  margin-left: 0;
+  color: #6070ff;
+}
+
+.tags ul {
+  display: flex;
+  list-style: none;
+  align-items: center;
+  padding: 0;
+  gap: 8px;
+  margin-left: 16px;
+}
+
+.tags ul li {
+  line-height: 16px;
+  letter-spacing: 3%;
+  background-color: #ebebff;
+  border-radius: 8px;
+}
+
+.action {
+  margin-left: 16px;
+  margin-bottom: 40px;
+  border: solid #6070ff;
+  width: 124px;
+  height: 48px;
+  padding: 12px;
+  font-weight: 500;
+  border-radius: 8px;
+  color: #396df2;
+}
+
+/* About section */
+
+.upper-part {
+  margin-left: 24px;
+  margin-right: 24px;
+}
+
+.about-myself {
+  max-width: 327px;
+  max-height: 101px;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  margin-bottom: 12px;
+  margin-top: 144px;
+}
+
+.about-text {
+  max-width: 327px;
+  max-height: 144px;
+  font-size: 16px;
+  margin-bottom: 12px;
+}
+
+.about-connect {
+  margin-bottom: 16px;
+  color: #7f8cff;
+}
+
+.about-tags ul {
+  display: flex;
+  list-style: none;
+  gap: 21px;
+  padding-left: 11px;
+}
+
+.resume {
+  height: 48px;
+  width: 162px;
+  left: 375px;
+  top: 4769px;
+  border-radius: 8px;
+  margin-top: 34px;
+  padding: 12px;
+  color: #396df2;
+  margin-bottom: 60px;
+  border: 1px solid #6070ff;
+}
+
+.languages {
+  margin-left: 36px;
+  display: flex;
+  margin-bottom: 36px;
+}
+
+.lang {
+  width: 237px;
+  height: 24px;
+  margin-right: 26px;
+  font-size: 20px;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+}
```

Comment:
- [ ] Please give your languages a margin top and bottom to match the figma design and look interactive and also kindly reduce margin to the left.
## your design 👇
![skillst](https://user-images.githubusercontent.com/66992901/193405143-8705be20-e96f-4b02-8cc4-e6c6293ecd1b.PNG)

## Figma design👇
![skillsfigma](https://user-images.githubusercontent.com/66992901/193405169-9ba9d251-31aa-4ed9-ae13-652e79e108eb.PNG)